packages/core/src/components/Table/TableRow/TableRow.tsx

```diff
@@ -14,26 +15,67 @@ export interface ITableRowProps extends VibeComponentProps {
   highlighted?: boolean;
   children?: React.ReactElement<ITableCellProps> | React.ReactElement<ITableCellProps>[];
   style?: React.CSSProperties;
+  expandableRowRenderer?: () => React.ReactNode;
+  onClick?: (event: MouseEvent) => void;
 }
 
 const TableRow: VibeComponent<ITableRowProps, HTMLDivElement> = forwardRef(
-  ({ highlighted, children, style, id, className, "data-testid": dataTestId }, ref) => {
+  (
+    {
+      highlighted,
+      children,
+      style,
+      id,
+      className,
+      "data-testid": dataTestId,
+      onClick: propsOnClick = () => {},
```

Comment:
1. Casting because there is already an `onClick` and i find it more readable than naming one of the `onClick` and the other one `handleClick` (or vice versa).
Do you want me to rename it?
2. The default value is for avoiding an explicit checking of the `propsOnClick` value.
If `propsOnClick` won't be sent (which is something that might be happening),  the `propsOnClick(event)` won't crash due to the fact that we have a default value of empty function